### PR TITLE
Build: Moved core and themes resource in their own subfolders

### PR DESCRIPTION
**Pull request by LaurentGoderre**
Partially implements #6809

### CODE REVIEW

**Comment by LaurentGoderre** [comment]
GTG!

File: site/includes/footerresources.hbs [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <!--[if gte IE 9 | !IE ]><!-->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/{{environment.jqueryVersion}}/jquery{{environment.suffix}}.js"></script>
-<script src="{{assets}}/js/wet-boew{{environment.suffix}}.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/{{environment.jqueryVersion}}/jquery.js"></script>
```
Comment (nschonni) [inline site/includes/footerresources.hbs]: It appears that jQuery is always server unminified after this change

Comment (LaurentGoderre) [inline site/includes/footerresources.hbs]: Really? I thought I fixed that

Comment (LaurentGoderre) [inline site/includes/footerresources.hbs]: We would need to move the UseMinAssets to a separate task, it gets really annoying to update it on every theme.

**Pull request merged**